packages/fxa-settings/src/pages/CannotCreateAccount/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { routeName } from '.';
+import { screen, render } from '@testing-library/react';
+import { logPageViewEvent } from '../../lib/metrics';
+import CannotCreateAccount from '.';
```

Comment:
Q: merge w/ L6's `import { routeName } from '.';`?
(insert joke here about import ESLinting)

Reply:
Thank you for always checking our imports 🤦‍♀️